src/lib/lumi.ts

```diff
@@ -4671,9 +4836,11 @@ export const toZigbee = {
         },
     } satisfies Tz.Converter,
     lumi_curtain_limits_calibration: {
-        key: ['limits_calibration'],
+        key: ['limits_calibration', 'manual_calibration'],
```

Comment:
Why not just use `limits_calibration` instead of adding `manual_calibration`?

Reply:
I wanted a distinction between automatic and manual calibration, but never mind, I'll change it